samples/dnn/object_detection.py

```diff
@@ -54,15 +54,17 @@
                     dest='asyncN',
                     help='Number of asynchronous forwards at the same time. '
                          'Choose 0 for synchronous mode')
+parser.add_argument('--postprocessing', choices=['yolo', 'ssd', 'faster_rcnn', 'yolov8'],
+                    help='Optional name of AI algo. ')
```

Comment:
Not representative description. What about "Detection results post-processing kind depends on model topology"

Reply:
We have changed the description as above, thanks.